server.js

```diff
@@ -26,10 +46,49 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Hello! Try out the routes '/titles' or '/titles/id/:id' ");
+});
```

Comment:
I can see on your reference that you had "dotenv" but are not using it here.
I recommend it, I would add in this section:

app.get("/", (req, res) => {
res.send({ "Routes": listEndpoints(app)});
});

Dont forget to add this: dotenv.config()
somewhere below the imports